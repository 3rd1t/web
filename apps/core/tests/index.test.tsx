import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"
import Index from "../pages/index"

describe(" Index", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Index />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
