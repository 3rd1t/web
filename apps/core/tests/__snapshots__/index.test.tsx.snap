// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Index renders correctly 1`] = `
<div
  className="flex flex-col items-center w-full"
>
  <div
    className="space-y-4 text-left sm:text-center md:space-y-8 xl:space-y-12 lg:text-left"
  >
    <h2
      className="text-4xl font-extrabold leading-10 tracking-tight text-gray-900 xl:text-5xl sm:leading-none md:text-5xl"
    >
      <p>
        Hello World
      </p>
    </h2>
    <div
      className="text-gray-600 sm:text-lg sm:mx-auto md:text-xl lg:mx-0"
    >
      very important text goes here
    </div>
    <div
      className="space-y-4 md:space-x-4 md:space-y-0"
    >
      <div
        className="sm:flex sm:justify-center lg:justify-start"
      >
        <div
          className="rounded-md shadow"
        >
          <a
            className="border border-transparent rounded-sm md:py-4 md:text-lg md:px-10 flex items-center justify-center w-full px-8 py-3 text-base font-medium leading-6 transition duration-150 ease-in-out focus:outline-none text-white bg-red-900 hover:bg-red-800 hover:text-red-100"
            href="#"
          >
            Primary
          </a>
        </div>
        <div
          className="mt-3 sm:mt-0 sm:ml-3"
        >
          <a
            className="border border-transparent rounded-sm md:py-4 md:text-lg md:px-10 flex items-center justify-center w-full px-8 py-3 text-base font-medium leading-6 transition duration-150 ease-in-out focus:outline-none text-red-700 bg-red-100 hover:text-red-600 hover:bg-white"
            href="Secondary"
          >
            Secondary
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
